lib/rollbax/notifier.ex

```diff
@@ -44,4 +44,10 @@ defmodule Rollbax.Notifier do
     %{level: Keyword.get(config, :level, :error),
       metadata: Keyword.get(config, :metadata, [])}
   end
+
+  defp ts_to_unix({{_yy, _mm, _dd} = date, {h, m, s, _ms}}) do
```

Comment:
We have four positions for a year value, let's go with `{{_, _, _} = date, {hour, min, sec, _}}`.